scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/LegacyScanner.scala

```diff
@@ -695,10 +695,13 @@ class LegacyScanner(input: Input, dialect: Dialect) {
         setStrVal()
         token = STRINGLIT
       }
-    } else if (ch == '\\') {
-      nextRawChar()
+    } else if (ch == '\\' && !multiLine) {
       putChar(ch)
       nextRawChar()
+      if (ch == '"' || ch == '\\') {
```

Comment:
the linked compiler code looks like this:
```
        if (ch == '"' || ch == '\\')
          putChar(ch)
          nextRawChar()
```

is it using significant indentation without `then`? or is `nextRawChar()` outside of the `if` block?

Reply:
yep - significant indentation without then